src/middlewares/auth.js

```diff
@@ -34,7 +34,7 @@ class Authentication {
    *
    * @returns {Object} Object
    */
-  static async isAdmin(req, res, next) {
+  static isAdmin(req, res, next) {
```

Comment:
I see you removed the async functionality from `isAdmin` and `authenticate` methods. Is there a reason why?

Reply:
Thank you for the feedback. I took the async functionality out since that could lead to a leak. After testing my app. I notice that the careless use of ```await``` could move functionality to the next middleware without a proper token authentication check. Making the process synchronous ensured that never happened. Thank you

Reply:
Alright. Cool!.